src/lib.rs

```diff
@@ -1,7 +1,87 @@
-//! # HoneyBadgerBFT
+//! # Honey Badger BFT
 //!
-//! Library of asynchronous Byzantine fault tolerant consensus known as "the
-//! honey badger of BFT protocols" after a paper with the same title.
+//! An implementation of [The Honey Badger of BFT Protocols](https://eprint.iacr.org/2016/199.pdf),
+//! an asynchronous, Byzantine fault tolerant consensus algorithm.
+//!
+//! This crate only implements the protocol itself and is meant to be instantiated in a number of
```

Comment:
I have a number of recommendations for rearranging and consolidating the text - I will send an email and we can decide the best way to proceed with these types of changes. I don't want to clutter with a bunch of comments, but that may be the preferred method.